src/message.c

```diff
@@ -4071,7 +4070,11 @@ msg_warn_missing_clipboard(void)
 {
     if (!global_busy && !did_warn_clipboard)
     {
-       msg(_(warn_clipboard));
+#ifdef FEAT_CLIPBOARD
+       msg(_("W23: Clipboard register not available, using register 0"));
+#else
+       msg(_("W24: Clipboard register not available"));
```

Comment:
I think this should be a little more specific about the fact that vim doesn't have this functionality compiled with ideally a link to a doc detailing what is `+clipboard`. Mainly for new users that they will likely now about `+clipboard` and the fact that you need to recompile vim.

Probably something like: "Clipboard register is not available in this build of vim. To learn more read `:h clipboard`"